purchase_order_supplier_phone/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     "website": "https://www.quartile.co",
     "category": "Purchase",
     "license": "LGPL-3",
-    "depends": ["purchase"],
+    "depends": ["purchase_ext_sst"],
```

Comment:
To make the reason of dependency clear.
```suggestion
    "depends": [
        "purchase_ext_sst",  # is_default_partner()
    ],
```